src/Mvc/Mvc.Core/src/ModelBinding/FormValueProvider.cs

```diff
@@ -13,7 +13,10 @@ namespace Microsoft.AspNetCore.Mvc.ModelBinding;
 /// </summary>
 public class FormValueProvider : BindingSourceValueProvider, IEnumerableValueProvider
 {
+    internal const string CultureInvariantFieldName = "__Invariant";
```

Comment:
_**nit**_: I have no suggestions but would be great if it could be in another shared place instead of here. We are already using it in 3 different places

Reply:
Maybe in `HtmlHelperOptions`? We already have `_idAttributeDotReplacement` there.

Reply:
Actually, that won't work because `HtmlHelperOptions` is not accessible from `Mvc.Core`.

I wonder if we would want this string to be configurable, though? `IdAttributeDotReplacement` in `HtmlHelperOptions`, for example, can be overridden.

Reply:
Let's wait for someone to file an issue rather than proactively making things harder for ourselves.

Reply:
We determined in API review that it's not necessary to make the "__Invariant" string configurable.

There doesn't seem to be another good existing location to place this constant, so how about a new helper class, `FormValueHelper`?

Reply:
The string `__Invariant` exists here, once in src/Mvc/Mvc.TagHelpers/test/InputTagHelperTest.cs, and in generated HTML files. Am I missing something❔ If not, I suggest leaving things as-is.